static/app/views/settings/projectDebugFiles/index.tsx

```diff
@@ -59,7 +59,7 @@ class ProjectDebugSymbols extends AsyncView<Props, State> {
         {
           query: {
             query: location.query.query,
-            file_formats: ['breakpad', 'macho', 'elf', 'pe', 'pdb', 'sourcebundle'],
+            // file_formats: ['breakpad', 'macho', 'elf', 'pe', 'pdb', 'sourcebundle'],
```

Comment:
@matejminar you seem to have added this query, but it hides wasm and now also bcsymbolmap and plist files.  is there any harm removing this part of the query?  i've tested this locally and on my small example it seemed to work fine.

Reply:
It is there so that we don't show proguard mappings in this list.
They are displayed in a different place:https://github.com/getsentry/sentry/blob/9707fd3f98fa51d5d363cbe34b59aa51588db246/static/app/views/settings/projectProguard/projectProguard.tsx#L50